src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_variantContentPricing.html.twig

```diff
@@ -1,31 +1,29 @@
 {% import "@SyliusAdmin/Common/Macro/money.html.twig" as money %}
 
-<div class="ui segment">
-    <div class="ui small header">
-        {{ 'sylius.ui.pricing'|trans }}
-    </div>
+<h4 class="ui top attached large header">{{ 'sylius.ui.pricing'|trans }}</h4>
+<div class="ui attached segment">
     <table class="ui very basic celled table">
         <tbody>
-        <tr>
-            <td class="gray text"><strong>{{ 'sylius.ui.channels'|trans }}</strong></td>
-            <td class="gray text"><strong>{{ 'sylius.ui.price'|trans }}</strong></td>
-            <td class="gray text"><strong>{{ 'sylius.ui.original_price'|trans }}</strong></td>
-        </tr>
-        {% set currencies = sylius_channels_currencies() %}
-        {% for channelPricing in variant.channelPricings %}
-            <tr class="pricing">
-                <td class="five wide gray text">
-                    <strong>{{ channelPricing.channelCode }}</strong>
-                </td>
-                {% set channelCode = channelPricing.channelCode %}
-                <td>{{ money.format(channelPricing.price, currencies[channelCode]) }}</td>
-                {% if channelPricing.originalPrice != null %}
-                    <td>{{ money.format(channelPricing.originalPrice, currencies[channelCode]) }}</td>
-                {% else %}
-                    <td><span class="gray text">-</span></td>
-                {% endif %}
+            <tr>
+                <td class="gray text"><strong>{{ 'sylius.ui.channels'|trans }}</strong></td>
+                <td class="gray text"><strong>{{ 'sylius.ui.price'|trans }}</strong></td>
+                <td class="gray text"><strong>{{ 'sylius.ui.original_price'|trans }}</strong></td>
             </tr>
-        {% endfor %}
+                {% set currencies = sylius_channels_currencies() %}
+                {% for channelPricing in variant.channelPricings %}
+                    {% set channel = channelPricing.channelCode|sylius_channel %}
+                    <tr class="pricing">
+                        <td>{{ channel.name }}</td>
+                        <td>{{ money.format(channelPricing.price, currencies[channel.code]) }}</td>
+                        <td>
+                            {% if channelPricing.originalPrice != null %}
+                                {{ money.format(channelPricing.originalPrice, currencies[channel.code]) }}
+                            {% else %}
+                                <span class="gray text">-</span>
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
```

Comment:
The table in this file and in `src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_pricing.html.twig` is almost the same - maybe we should extract it to some common template? 🤔 We would need to pass only a for loop array as its data. Of course, we can do it in the separate PR, but I fear it would vanish in the doom of abyss of other issues 😄  🕳

Reply:
I  opened new PR with fixes of your comment :) https://github.com/Sylius/Sylius/pull/10772